### PR TITLE
normalizeDepiction() should always center coordinates on the coordinate centroid, irrespective of the canonicalize parameter

### PR DESCRIPTION
**Pull request by ptosco**
I had not noticed this so far as centering takes place anyway before displaying structures `MolDraw2D`, but this should be corrected as `normalizeDepiction` should always center and rescale irrespective of the `canonicalize` parameter value.

### CODE REVIEW

File: Code/GraphMol/Depictor/catch_tests.cpp [hunk 1]
```diff
@@ -2272,3 +2272,68 @@ M  END
...
+  2 25  1  0
+  6  2  1  0
+  8  4  1  0
+ 19 14  1  0
+M  END)CTAB"_ctab;
+  REQUIRE(m);
+  RDDepict::normalizeDepiction(*m, -1, 0);
+  auto ctd = MolTransforms::computeCentroid(m->getConformer());
+  CHECK(fabs(ctd.x) < 1.0e-4);
+  CHECK(fabs(ctd.y) < 1.0e-4);
```
Comment (greglandrum) [inline Code/GraphMol/Depictor/catch_tests.cpp:2338]: I think the idiomatic way to do this catch2 is:
```suggestion
  CHECK_THAT(ctd.x, Catch::WithinAbs(0, 1e-4));
  CHECK_THAT(ctd.y, Catch::WithinAbs(0, 1e-4));
```

Comment (ptosco) [reply Code/GraphMol/Depictor/catch_tests.cpp]: Done, thanks.

Comment (greglandrum) [reply Code/GraphMol/Depictor/catch_tests.cpp]: @ptosco apologies that I didn't actually test that suggestion. The CI failures. It's probably actually `Catch::Matchers::WithinAbs`

Comment (ptosco) [reply Code/GraphMol/Depictor/catch_tests.cpp]: No worries, I'll fix it now.

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**